src/pages/Internship.jsx

```diff
@@ -0,0 +1,181 @@
+{/* w-1440 3/4 */}
+{/*  */}
+
+import React from "react";
+import Footer from "../components/Footer";
+import Instagram from '../images/Instagram.png'; 
+import fam_placeholder from '../images/fam_placeholder.png'
+import TotoPic from '../images/TotoPic.png'
+
+
+const Internship = () => {
+    return (
+        <div className="flex flex-col"> {/* parent container */}
+            <header className="bg-[#cbc7b7] h-[840px] w-[1440px] flex flex-col items-center justify-center">  
```

Comment:
my only issue is the spacing, keeping it w-[1440px] on bigger screens makes it not go the full width like mine, just make it w-full and do it on the other occurences as well